### PR TITLE
[DOCFIX] Improve on hive doc

### PR DESCRIPTION
**Pull request by jiacheliu3**

### CODE REVIEW

File: docs/en/compute/Hive.md [hunk 1]
```diff
@@ -268,7 +268,7 @@ hive> alter table u_user set location "alluxio:///ml-100k";
 Since Alluxio 2.0, one can directly use Alluxio HA-style authorities in Hive queries without any configuration setup.
 See [HA authority]({{ '/en/deploy/Running-Alluxio-On-a-HA-Cluster.html' | relativize_url }}#ha-authority) for more details.
 
-### Experimental: Use Alluxio as the Default Filesystem
```
Comment (jiacheliu3) [inline docs/en/compute/Hive.md:271]: This line was from 2018. I guess not "experimental" any more?

Comment (LuQQiu) [reply docs/en/compute/Hive.md:271]: Agree that we can remove this word

**Review by LuQQiu — APPROVED** [review]
LGTM, thanks!

File: docs/en/compute/Hive.md (hunk 1 above)
Comment (jiacheliu3) [reply docs/en/compute/Hive.md]: I talked to Gene, seems this is still "experimental". So i brought it back.

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by jiacheliu3** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**